src/main/js/config/GlobalConfig.tsx

```diff
@@ -50,24 +50,33 @@ const GlobalConfig: FC<Props> = ({ link }) => {
   return (
     <ConfigurationForm
       isValid={formState.isValid}
-      isReadonly={isReadonly}
+      isReadOnly={isReadOnly}
       onSubmit={handleSubmit(update)}
       {...formProps}
     >
       <Title title={t("scm-repository-mirror-plugin.settings.title")} />
       <Checkbox
         label={t("scm-repository-mirror-plugin.form.httpsOnly.label")}
         helpText={t("scm-repository-mirror-plugin.form.httpsOnly.helpText")}
-        disabled={isReadonly}
+        disabled={isReadOnly}
         {...register("httpsOnly")}
       />
+      <hr />
+      <h2 className="subtitle">{t("scm-repository-mirror-plugin.form.verificationFilters")}</h2>
```

Comment:
I'm missing a global 'ff only' flag mirroring the repository specific one.

Reply:
Good point, it should be trivial to include this as it is already part of the dto and filter merge in the store.